src/BlazorWebView/src/Maui/Android/AndroidWebKitWebViewManager.cs

```diff
@@ -88,19 +88,6 @@ internal void SetUpMessageChannel()
 			_webview.PostWebMessage(new WebMessage("capturePort", destPort), AndroidAppOriginUri);
 		}
 
-		protected override async ValueTask DisposeAsyncCore()
-		{
-			await base.DisposeAsyncCore();
-
-			if (_nativeToJSPorts is not null)
-			{
-				foreach (var port in _nativeToJSPorts)
-				{
-					port?.Close();
-				}
-			}
-		}
-
```

Comment:
Turns out that these ports get closed automatically when the WebView gets unloaded. Now that this logic was running for the first time, an exception was being thrown because the ports were being closed twice. Hence the removal of this method.